src/collision/broad_phase.rs

```diff
@@ -200,7 +202,11 @@ fn sweep_and_prune(
                 continue;
             }
 
-            broad_collision_pairs.push((*ent1, *ent2));
+            if *ent1 < *ent2 {
+                broad_collision_pairs.push((*ent1, *ent2));
+            } else {
+                broad_collision_pairs.push((*ent2, *ent1));
+            }
```

Comment:
Too bad [`minmax`](https://doc.rust-lang.org/stable/std/cmp/fn.minmax.html) is nightly only :/